source/guides/maintenance/fix-broken-install.rst

```diff
@@ -75,10 +76,11 @@ Mount root partition, verify, and fix
 
    :ref:`Learn more about how swupd works <swupd-guide>`.
 
-#. After the process is complete, unmount the root partition:
+#. After the process is complete, unmount the boot and root partitions:
```

Comment:
swupd does not manage the EFI partition. So we it's not necessary to mount the boot partition.